airflow/example_dags/tutorial_etl_dag.py

```diff
@@ -108,6 +110,7 @@ def load(**kwargs):
     transform_task = PythonOperator(
         task_id='transform',
         python_callable=transform,
+        op_args=[extract_task.output['order_data']],
```

Comment:
If I'm not mistaken we would like to keep the old way in this example because we use it to compare the old vs. new approach. CC @vikramkoka ?

Reply:
Correct. This is dag is to show the classic way, and tutorial_taskflow_api_etl.py is the companion showing the new way.

Reply:
Yes, that is correct. This was intended to show the "classic way".